docs/core/whats-new/dotnet-10/sdk.md

```diff
@@ -202,3 +202,31 @@ name = "Microsoft.Testing.Platform"
 ```
 
 For more details, see [Testing with `dotnet test`](../../testing/unit-testing-with-dotnet-test.md).
+
+## Preview 7 additions
+
+The following feature was added in Preview 7:
+
```

Comment:
Remove this header so that the Preview 7 features are incorporated into the main article.